src/styles/variables/colors.scss

```diff
@@ -51,5 +51,8 @@ $color-shadow-alpha-15: rgba(0, 33, 87, 0.15);
 $color-shadow-alpha-10: rgba(0, 33, 87, 0.15);
 
 // General Colors
-$color-black: #000000 !default;
-$color-white: #ffffff !default;
+$color-black:   #000000 !default;
+$color-white:   #ffffff !default;
+$color-red:     $color-error !default;
```

Comment:
It is clearer to define `$color-error` as `$color-red` and `$color-red` as `#FF0D6A`, because later, when we will decide to change `$color-error` to green or so, it won't break the value of `$color-red` variable.

Reply:
updated